db/seeds.rb

```diff
@@ -12,103 +12,120 @@
                              description: "This dish is so good you'll feel like prince charming just rode in to save the day atop his trusty white stallio...er...unicorn",
                              price: 10.50,
                              status: "active",
-                             categories: [entree_cat])
+                             categories: [entree_cat],
+                             image: File.open('public/UnicornOnTheCob.jpg'))
 
 medusa         = Item.create(title: "Medusa Hair Pasta",
                              description: "Freshly prepared by Perseus, our world-renowned and legendary (be)head chef",
                              price: 12.99,
                              status: "active",
-                             categories: [pasta_cat, entree_cat])
+                             categories: [pasta_cat, entree_cat],
+                             image: File.open('public/MedusaHairPasta.jpg'))
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.